deploy.sh

```diff
@@ -140,7 +140,7 @@ function retry_terraform {
 
 function setup_terraform {
   TF_LATEST=$(curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep tag_name | cut -d'"' -f4)
-  if which terraform > /dev/null; then
+  if which terraform 2> /dev/null; then
```

Comment:
I had removed it on purpose. Any issue here? It was display the path on console. If this is not working then check return code instead please.

Reply:
without 2 it gives a huge verbose when the binary is not present in the path the very first time. I Will check if there is any other alternative.

Reply:
Not sure what verbose you are talking about.. on my linux:
 ```
# if which terraform > /dev/null; then echo test; fi
test
# if which terraform 2> /dev/null; then echo test; fi
/usr/local/bin/terraform
test
```
Missing in PATH:
```
# if which terraform1 2> /dev/null; then echo test; fi
# if which terraform1 > /dev/null; then echo test; fi
```